samples/Rewrite/ConditionParser/Condition.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Rewrite.ConditionParser
+{
+    public class Condition
+    {
+        public List<ConditionTestStringSegment> TestStringSegments { get; set; }
+        public GeneralExpression ConditionRegex { get; set; }
+        public List<string> Flags { get; set; }
```

Comment:
If this is a closed set of values it should be parsed before adding it to the condition. Try an enum or class.